syntaxes/omt.tmLanguage.json

```diff
@@ -31,6 +31,13 @@
                         "3": { "patterns": [{ "include": "#_comments-only" }] }
                     }
                 },
+                {
+                    "match": "(?:\\G|^) +(promoteSubMenuItemToMainMenu): *(\\w*)(.*)$",
```

Comment:
The capturing for the last group needs to be added, to enable highlighting just comments at the end of the line.

Reply:
@emielb i think i added the write part with comments only, let me know if it's ok now

Reply:
👍